### ISSUE TITLE
V1: Transaction count in blocks not counted accurately. Off by one or two depending on backend used

### ISSUE THREAD

**Issue by rbrooklyn** [mempool/mempool#27]
When using bitcoind backend, the transaction count is always off by two.

So, for example, block 617544 shows 2919 transactions:
![image](https://user-images.githubusercontent.com/11929484/74595383-91a32400-5038-11ea-8789-8a377d646b91.png)

In electris backend, it comes through as 2920 transactions, so is off by one:
![image](https://user-images.githubusercontent.com/11929484/74595389-a089d680-5038-11ea-9402-945335af3a62.png)

However, both numbers are "wrong", as the true number is 2921 according to every explorer I checked:
[Smartbit.com.au](https://www.smartbit.com.au/block/617544)
[blockchain.com](https://www.blockchain.com/btc/block/00000000000000000002216a97ef780f2a926c43cb3314eb0f69c36016187548)
[blockcypher.com](https://live.blockcypher.com/btc/block/00000000000000000002216a97ef780f2a926c43cb3314eb0f69c36016187548/?__cf_chl_jschl_tk__=5b529915439b247f8fe5fe39fa3c3b18be28b48f-1581801549-0-AY4ESOpQ34OFol4s7hIZowe3ncy2OA47FZUkK-G2gF7n8Lr8JstHn7hhXngf6C0UYkQKoqFfxpV4yn7q7SbDZvytvsGlW9TXX2WWVmxuO9WJnBTikB8HMJx8hvfZyXCDKoAZLxC6VCeTdUiEevoBY_vp4WEJEXj6LsbAeqnX_YBK3MsSac8z0xScVPYmyqrSnIJwxrkzGylT-j3BHzdrsxx1wbi5z_byuAfI7U-DQ-O3tdhbmmng1ON-PNMULOuBqFGZNa1wPm9sLza_nEU9CQHzuCthLI8ieUh8BuRjh4TxAtoNkoxnCU93eYGjYZ6wXn2-Tx5ZGRlWWr2U_C8kWNl2Uo2dIhA-EqNHn21DUBBsA1gM6Pbj-ydonAjU9NMZal1OlujaFI0Gj9dpZ-9nBqyZTymSkfbTP4tna1hT5zzl)

**Comment by wiz** [comment]
That's why we have a blockchain, to resolve this inconsistency between nodes :)

**Comment by softsimon** [comment]
Will investigate and fix :)

**Comment by rbrooklyn** [comment]
The backend _might_ not be the issue. Unsure of why I was getting a "off by two" error yesterday. I was messing around with my local copy of mempool.space, so it's _possible_ I broke something or maybe something is broken in the latest git?

This morning I reverted to a older version from about a month ago. Now, regardless of backend used, it's just off by one today (I use bitcoind, I'm assuming the mempool.space website is using electrs). So I'm thinking it's possible the coinbase transaction not being counted and it might be that "simple".

Sorry for the confusion.
